siteimprove/admin/js/siteimprove-gutenberg.js

```diff
@@ -25,7 +25,7 @@ var RecheckButton = function () {
         },
         onClick: function () {
             setClicked(true);
-            if (typeof window.siteimprove !== "undefined") {
+            if (typeof window.siteimprove.recheck !== "undefined") {
```

Comment:
can you please test that it works like this 🙏

Reply:
Yes, it works. 

I've tested this by using a version of the plugin that didn't have: 

```
    recheck: function (url, token, callback) {
      this.callback = callback;
      this.url = url;
      this.token = token;
      this.method = "recheck";
      this.common(url);
    },
```
As part of siteimprove.js, and it worked as intended and put out the message in the log.